### PR TITLE
Add PR Section to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pkramme**
I added a section about pull requests, please inform me whether it's good or not.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,3 +34,7 @@ and are better then 'this thing there is broken. go fix it'. It also helps if yo
 This is the place to disguss. Don't get angry if someone don't share your opinion, rather try to convince him while staying friendly. This is the most important thing: If your unfriendly no matter your skill level, no one will respect you. Always remember that you're the one who wants something.
 ### Reactions
 Software projects are mostly driven by community feedpack and ideas, so if you read another issue that wants to add a feature, react to it by writing something like "I like / don't like it" or by using the emoticons to show the creators what you want. This is the most valuable feedpack a developer can have. 
+## Pull Requests
+In order to use a create a Pull Request, you have to create a fork first. To optain infos about how to get your fork working check out [this](https://github.com/gitskarios/Gitskarios/blob/develop/FORK.md). If you still have problems getting your branch working, create an issue explaining what your problem is.
```
Comment (alorma) [inline CONTRIBUTING.md]: To use **a** create ?? should it be **or**?

Comment (alorma) [inline CONTRIBUTING.md]: Obtain

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,3 +34,7 @@ and are better then 'this thing there is broken. go fix it'. It also helps if yo
...
 ### Reactions
 Software projects are mostly driven by community feedpack and ideas, so if you read another issue that wants to add a feature, react to it by writing something like "I like / don't like it" or by using the emoticons to show the creators what you want. This is the most valuable feedpack a developer can have. 
+## Pull Requests
+In order to use a create a Pull Request, you have to create a fork first. To optain infos about how to get your fork working check out [this](https://github.com/gitskarios/Gitskarios/blob/develop/FORK.md). If you still have problems getting your branch working, create an issue explaining what your problem is.  
+When you're about creating a pull request, you have to name it after it contains, for example: You created feature X, then name it after this feature. You created a fix for a issue, name it 'Fix #tag-of-the-issue' so everyone knows what you were doing.  
+**ALSO KEEP IN MIND THAT PULL REQUEST WITH A BASE BRANCH OF MASTER OR DEVELOP WILL NOT BE ACCEPTED. CREATE ONE BRANCH FOR ONE FIX OR FEATURE.**
```
Comment (alorma) [inline CONTRIBUTING.md]: Add a section so branch name have a meaningfull name, i won't accept `branch`, `patch-1`, etc...

Comment (pkramme) [inline CONTRIBUTING.md]: hehe whooops, my last pr...

**Pull request merged**